config/locales/client.en.yml

```diff
@@ -2673,6 +2673,7 @@ en:
         create: "Sorry, there was an error creating your post. Please try again."
         edit: "Sorry, there was an error editing your post. Please try again."
         upload: "Sorry, there was an error uploading that file. Please try again."
+        upload_error_zero: "Sorry, there was an undetermined error uploading that file. Please try again, or check your browser's developer console for more information."
```

Comment:
I feel like this is too technical for the average user. I suggest we use the error right above it `upload`